src/tests/golden/testdata/embeddedrules/callSimplify.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+/**
+ * @param  mixed[]  $array
```

Comment:
nit: there is an extra space between `mixed[]` and `$array`.

Reply:
Done.